components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAOTest.java

```diff
@@ -605,13 +605,13 @@ public void testGetAppInformationByAppNameWithExceptions(Boolean enablePKCE) thr
         mockStatic(OAuth2ServiceComponentHolder.class);
         when(OAuth2ServiceComponentHolder.isPkceEnabled()).thenReturn(enablePKCE);
 
-        TestUtils.initPrivilegedCarbonContext("PRIMARY", -1234, USER_NAME);
+        CommonTestUtils.initPrivilegedCarbonContext("PRIMARY", -1234, USER_NAME);
```

Comment:
Better to use `carbon.super` instead of `PRIMARY` since the parameter is `tenantDomain`. Or we can define another method in the CommonTestUtils with a different signature where we have only the `username`?